raster/src/test/scala/geotrellis/raster/render/png/RenderPngTests.scala

```diff
@@ -339,7 +339,7 @@ class RenderPngTests extends AnyFunSuite with Matchers with TileBuilders with Ra
     val stops = Array(10000, 20000, 30000, 40000, 50000, 60000, 70000)
     val colorMap = ColorMap(stops, ramp)
 
-    val greyPng = colorMap.render(tile).map(z => z >> 8 & 0xFF).renderPng(GreyPngEncoding(0x00))
+    val greyPng = colorMap.render(tile).map(z => z >> 8 & 0xFF).renderPng(GreyPngEncoding)
```

Comment:
Good question why it worked, technically it was written as an `greya` image that could be considered by some readers as grey due to a colorType set to 0 in header.